common/src/main/java/com/wynntils/wynn/model/territory/GuildTerritoryModel.java

```diff
@@ -70,10 +72,8 @@ public static void onAdvancementUpdate(AdvancementUpdateEvent event) {
 
             if (territoryProfile == null) continue;
 
-            newMap.put(entry.getKey(), new TerritoryPoi(territoryProfile, entry.getValue()));
```

Comment:
Is there anyway we can remove TerritoryPoi now? Or maybe that is not relevant?

Reply:
It is a class used for rendering on map